client/server/bundler/index.js

```diff
@@ -31,11 +31,7 @@ function middleware( app ) {
 	app.set( 'compiler', compiler );
 
 	if ( shouldProfile ) {
-		compiler.apply(
-			new webpack.ProgressPlugin( {
-				profile: true,
-			} )
-		);
+		new compiler.webpack.ProgressPlugin( { profile: true } ).apply( compiler );
```

Comment:
Did this work? `compiler` doesn't have an `.apply` method, does it?

Reply:
I never used it before tbh. I was surprised about that `compile.apply()` as well...